packages/react/src/primitives/utils/testUtils.ts

```diff
@@ -1,25 +1,23 @@
-import * as React from 'react';
-export const useTestId = (testId: string, component: string) => {
-  const newTestId = React.useMemo(
-    () => (testId && component ? `${testId}-${component}` : undefined),
-    [testId, component]
-  );
-  return newTestId;
-};
```

Comment:
Since `useTestId` always returns a string, there is no need to memoize as the reference will remain the same unless the arguments are updated (which would result in `useMemo` recomputing)